sbpack/noncwl/manifest.py

```diff
@@ -0,0 +1,271 @@
+from typing import Union, Optional
+from sevenbridges.models.project import Project
+
+import logging
+import sbpack.lib as lib
+import argparse
+import os
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def paths_to_check(file_name: Union[str, list]) -> list:
+    if ":" in file_name:
+        return list()
+
+    if ";" in file_name:
+        chk = []
+        for file in file_name.split(";"):
```

Comment:
what if file is a list? is that possible here?

Reply:
Good point. I have reworked this to only accept string inputs.